pwa/src/components/common/arrayInput.tsx

```diff
@@ -4,7 +4,7 @@ import * as _ from "lodash";
 import { deleteElementFunction, addElement } from "./elementCreation";
 
 interface ArrayInputProps {
-  data: Array<Record<"value", any>>;
+  data: any;
```

Comment:
Als vuistregel: nooit `any` gebruiken.

Note: misschien dat dit een scenario is waar het wél echt moet, heb ik niet verder getest.

Reply:
Nieuwe `elementCreation` staat op `development`, kan je een ticket voor jezelf aanmaken dat je een refactor door de hele applicatie doet waarbij je `ArrayInput` verwijdert waar mogelijk?